test/dummy/db/schema.rb

```diff
@@ -56,8 +55,9 @@
     t.datetime "created_at", precision: 6, null: false
     t.datetime "updated_at", precision: 6, null: false
     t.text "arguments"
-    t.index ["task_name", "created_at"], name: "index_maintenance_tasks_runs_on_task_name_and_created_at"
     t.integer "lock_version", default: 0, null: false
+    t.index ["task_name", "created_at"], name: "index_maintenance_tasks_runs_on_task_name_and_created_at", order: { created_at: :desc }
```

Comment:
@etiennebarrie you'd mentioned no longer needing this index from the changes in this PR, but I'm wondering why. Won't `TaskData#completed_runs` still use this index to show historical run data?

@promulo did you add the order back manually here, or were you able to rerun the migrations to bring this back?

Reply:
@adrianna-chang-shopify The order was added by the [first migration](https://github.com/Shopify/maintenance_tasks/blob/main/db/migrate/20201211151756_create_maintenance_tasks_runs.rb#L20) (after `rails db:reset db:migrate`). It was removed manually by https://github.com/Shopify/maintenance_tasks/commit/8cbe81099630c47c114b9e6e0dc05b7fa33fb4dd but there's no migration changing this index.